src/exception.rs

```diff
@@ -23,10 +23,10 @@ pub enum TrapKind {
 }
 }
 
-/// This is equal to the type defined by the `TrapKind` enum.
-/// TODO: How can we reuse `TrapKind` directly?
-const EXCEPTION_SYNC: usize = 0;
-const EXCEPTION_IRQ: usize = 1;
+/// Equals to [`TrapKind::Synchronous`], used in exception.S.
```

Comment:
Fixed in passing.

Reply:
Oh, I forgot to modify it last time.

Thanks for fixing